sphinxcontrib/images.py

```diff
@@ -346,7 +348,8 @@ def setup(app):
     global DEFAULT_CONFIG
```

Comment:
This global declaration isn't needed, and it might be good to remove it while you're editing this file